helper/main.go

```diff
@@ -48,17 +48,18 @@ func getStatus(dataDirectory string) {
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "%s\n", err)
 	} else {
-		if dataDirectory != "" {
-			status.DataDirectory = dataDirectory
-		} else {
+		status.DataDirectory = dataDirectory
+		if dataDirectory == "" {
```

Comment:
I was wondering if this should read `if status.DataDirectory == ""` instead, which feels ever so slightly more consistent to me.

Reply:
Ah yes, good point.